services/src/main/java/org/keycloak/services/resources/admin/info/ServerInfoAdminResource.java

```diff
@@ -179,16 +179,18 @@ private void setThemes(ServerInfoRepresentation info) {
 
             for (String name : themeNames) {
                 try {
-                    Theme theme = session.theme().getTheme(name, type);
-                    ThemeInfoRepresentation ti = new ThemeInfoRepresentation();
-                    ti.setName(name);
+                    Theme theme = session.theme().getTheme(name, type, false);
+                    if (theme != null) {
```

Comment:
What about doing something like this instead of adding the overloaded method to ThemeManager:

```suggestion
                    Theme theme = session.theme().getTheme(name, type);
                    if (theme != null && theme.getName().equals(name)) {
```

Reply:
Updated as you suggested